setup/root_binaries/tank

```diff
@@ -82,7 +82,8 @@ else
    # so try and get the parent and call its tank script
    # the parent location is stored in a config file
    curr_platform=`uname`
-   if [[ "${curr_platform}" == MINGW32_NT* ]] || [[ "${curr_platform}" ==  CYGWIN_NT* ]];
+   if [[ "${curr_platform}" == MINGW64_NT* ]] || [[ "${curr_platform}" == MINGW32_NT* ]] ||
```

Comment:
minor polish comment but since you are in there, would be nice to maybe unify this code to either all use `${curr_platform}` or `$uname_os_str`